.rubocop.yml

```diff
@@ -1245,6 +1245,6 @@ RSpec/DescribedClass:
   Enabled: false
 
 RSpec/MultipleMemoizedHelpers:
-  Max: 15
+  Max: 25
```

Comment:
This should be done on the faulty test area ( spec/serializers/decidim/forms/user_answers_serializer_spec.rb )

Don't change linter configuration.

Reply:
I updated it because it's more in the [release/0.24-stable](https://github.com/decidim/decidim/blob/release/0.24-stable/.rubocop_ruby.yml#L1249) rubocop configuration on decidim/decidim. 
Shouldn't we be up to date with this repo?

Reply:
Yes we should.